requirements.txt

```diff
@@ -6,3 +6,4 @@ progress
 PyYAML
 lastversion
 coloredlogs
+bumpversion
```

Comment:
@bjk7119 ,
It seems unnecessary because bumpversion is not imported in the code.
And bumpversion is being installed directly in .github/workflows/publish-release.yml without using requirements.txt.

Reply:
@soimkim 
yes, you are right
bumpversion is not imported in FL Util and installed directly in .github/workflows/publish-release.yml to bump version to new version.
So I deleted this line at previous commit.